src/components/TododItem.tsx

```diff
@@ -0,0 +1,136 @@
+/* eslint-disable prettier/prettier */
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { useEffect } from 'react';
+import cn from 'classnames';
+import { ErrorMessage, FilteredStatus, Todo } from '../types/Todo';
+import { TempTodo } from './TempTodo/TempTodo';
+import { deleteTodos } from '../api/todos';
+
+interface Props {
+  todos: Todo[];
+  setTodos: (value: Todo[]) => void;
+  setCount: (value: number) => void;
+  filterValue: FilteredStatus;
+  onInputChange: () => void;
+  tempTodo: Todo | null;
+  setErrorMessage: (value: ErrorMessage) => void;
+  inputRef: React.MutableRefObject<HTMLInputElement | null>;
+  deletedTodoId: number[];
+  setDeletedTodoId: React.Dispatch<React.SetStateAction<number[]>>;
+}
+
+export const TodoItem = ({
```

Comment:
It's not `TodoItem`, you need to rename to `TodoList`
```suggestion
export const TodoList = ({
```